src/plugins/discover/public/context_awareness/profile_providers/log_document_profile/profile.test.ts

```diff
@@ -39,7 +39,7 @@ describe('logDocumentProfileProvider', () => {
       logDocumentProfileProvider.resolve({
         rootContext: ROOT_CONTEXT,
         dataSourceContext: DATA_SOURCE_CONTEXT,
-        record: buildMockRecord('logs-2000-01-01', {
+        record: buildMockRecord('another-index', {
```

Comment:
I switched the index on these tests because they always would have passed even if their functionality was broken due to the logs `_index` check.